server/database/error.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Class representing Exception/Error handling
+ */
+
+'use strict';
+
+class CustomException extends Error{
```

Comment:
Make specific error classes - ie.) DatabaseError

CustomException is somewhat vague. https://stackify.com/node-js-error-handling/